{"abstract":[{"text":"The underlying error that caused the error condition","type":"text"}],"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"underlyingError","kind":"identifier"},{"kind":"text","text":": "},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":"? { "},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}],"languages":["swift"]}],"kind":"declarations"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/amplify\/graphqlresponseerror\/underlyingerror"]}],"identifier":{"url":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/GraphQLResponseError\/underlyingError","interfaceLanguage":"swift"},"schemaVersion":{"minor":3,"patch":0,"major":0},"metadata":{"symbolKind":"property","role":"symbol","roleHeading":"Instance Property","modules":[{"name":"Amplify"}],"fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"underlyingError","kind":"identifier"},{"kind":"text","text":": "},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"text":"?","kind":"text"}],"title":"underlyingError","externalID":"s:7Amplify20GraphQLResponseErrorO010underlyingD0s0D0_pSgvp"},"sections":[],"hierarchy":{"paths":[["doc:\/\/amplify-swift.Amplify\/documentation\/Amplify","doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/GraphQLResponseError"]]},"kind":"symbol","references":{"doc://amplify-swift.Amplify/documentation/Amplify":{"url":"\/documentation\/amplify","type":"topic","title":"Amplify","role":"collection","identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify","kind":"symbol","abstract":[]},"doc://amplify-swift.Amplify/documentation/Amplify/GraphQLResponseError/underlyingError":{"identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/GraphQLResponseError\/underlyingError","role":"symbol","kind":"symbol","type":"topic","abstract":[{"type":"text","text":"The underlying error that caused the error condition"}],"url":"\/documentation\/amplify\/graphqlresponseerror\/underlyingerror","title":"underlyingError","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"underlyingError","kind":"identifier"},{"text":": ","kind":"text"},{"text":"Error","preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier"},{"text":"?","kind":"text"}]},"doc://amplify-swift.Amplify/documentation/Amplify/GraphQLResponseError":{"title":"GraphQLResponseError","kind":"symbol","abstract":[{"type":"text","text":"An error response from a GraphQL API"}],"url":"\/documentation\/amplify\/graphqlresponseerror","role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"GraphQLResponseError","kind":"identifier"}],"identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/GraphQLResponseError","navigatorTitle":[{"kind":"identifier","text":"GraphQLResponseError"}],"type":"topic"}}}